bedrock/firefox/templates/firefox/mobile-en.html

```diff
@@ -0,0 +1,242 @@
+{% endblock %}
+
+{% block page_title %}{{ _('Mobile browsers for iOS and Android') }} | {{ _('Firefox') }}{% endblock %}
+{% block page_desc %}{{ _('Download Firefox and Firefox Focus for mobile and tablet.') }}{% endblock %}
+{% block page_og_title %}{{ _('Get the mobile browser built for you, not advertisers') }} | {{ _('Firefox') }}{% endblock %}
+{% block page_og_desc %}{{ _('Check out Firefox again. It’s fast, private and on your side. For iOS and Android.') }}{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% block site_header %}
+  {% with hide_nav_download_button=True %}
+    {% include 'includes/protocol/navigation/index.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+  {% with current='mobile' %}
+    {% include 'firefox/includes/hub/sub-nav.html' %}
+  {% endwith %}
+
+  <main role="main">
+    <header class="mzp-c-hero mzp-t-firefox mzp-has-image t-hero-primary">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <h1 class="mzp-c-hero-title"><span class="c-wordmark t-firefox">{{ _('Firefox') }}</span></h1>
+
+          <div class="mzp-c-hero-desc">
+            <p>{{ _('The mobile browser built for you') }}</p>
+            <p>{{ _('Take Firefox speed and privacy with you everywhere.') }}</p>
+          </div>
+
+          <div class="mzp-c-hero-cta">
+            <div class="header-product-ctas">
+              <button type="button"
+                      class="mzp-c-button mzp-t-product js-mobile"
+                      id="get-firefox"
+                      data-product="firefox"
+                      data-button-name="Send to SMS Modal - Firefox Mobile">
+                      {{ _('Get Firefox Mobile') }}</button>
+            </div>
+
+            <div class="mobile-download-buttons-wrapper">
+              <ul class="mobile-download-buttons firefox" id="mobile-download-buttons-firefox">
+                <li class="android">
+                  {{ google_play_button(href='https://app.adjust.com/2uo1qc?campaign=desktop_xp&amp;adgroup=Google_Play_Store&amp;creative=badge_en', id='playStoreLink') }}
+                </li>
+                <li class="ios">
+                  <a id="appStoreLink" href="https://app.adjust.com/2uo1qc?campaign=desktop_xp&amp;adgroup=iOS_App_Store&amp;creative=badge_en&amp;fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8" data-link-type="download" data-download-os="iOS">
+                    <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <section class="c-private t-private mzp-t-firefox">
+      <div class="mzp-l-content">
+        <h2>{{ _('Stay invisible on the go with Private Browsing mode') }}</h2>
```

Comment:
This section talks only about Private Browsing mode, but Tracking Protection is now enabled by default on Firefox for iOS, which is significantly more private than our messaging here suggests? I'm not sure if this is the case for Android yet as I don't own a device, but worth double checking and/or calling out?

Reply:
Hmmmm. We were not trying to update the page content in this iteration. I think this is probably an important update to make but I don't want to block on this.